### ISSUE TITLE
Dmx model not starting with 1st or next Channel from previous model

### ISSUE THREAD

**Issue by ScobbyDoo** [xLightsSequencer/xLights#3636]
1: When creating new dmx model
Start channel goes to 512 by default

Expected behaviour: model should be starts with 1st channel OR next channel of previous model

2 : when i try to duplicate same dmx model with all configuration like 
channel size,
Channel name,
Channel colour

Start channel is not generating in next index number (as i mentioned in 1st case)

To generate this issue,
Create dmx model
Set channels to 8(may be High or low)
Copy paste that model & check starting channel

**Comment by AzGilrock** [comment]
Ok there must be more to this than what you reported.  I started with a blank show folder and added 3 Moving Heads and the channels numbers started at 1 and go up as expected.
<img width="560" alt="Screenshot 2022-12-20 at 6 29 40 PM" src="https://user-images.githubusercontent.com/2024669/208799271-0643d2b7-fd3d-4c96-b65e-718acc8ad76f.png">

**Comment by AzGilrock** [comment]
Ok if I add a Falcon controller and then start adding the Moving Heads I see they start at 511 but additional head do daisy chain to the first one.  Not sure why it works that way.  Maybe @keithsw1111  could chime in.

**Comment by AzGilrock** [comment]
And actually it has nothing to do with Moving heads....if I add an Arch model it also started at 511.  Not sure why we reserve the first 510 channels.

**Comment by AzGilrock** [comment]
Ok playing around more it appears the models get added to the first channel not assigned to a controller because we don't automatically assume which controller you want the model added to.  You add models and then map them to controllers either with parameters or the visualizer.  I unchecked the Auto Size box for the Falcon and then told it there were 5 universes.  Then when I added the first model it started at channel 2551.  So I'm thinking this is working as planned and they probably don't want to change anything.
